include/mbedtls/x509.h

```diff
@@ -385,8 +385,8 @@ int mbedtls_x509_time_is_future(const mbedtls_x509_time *from);
  * \param san      The target structure to populate with the parsed presentation
  *                 of the subject alternative name encoded in \p san_raw.
  *
- * \note           Only "dnsName" and "otherName" of type hardware_module_name
- *                 as defined in RFC 4180 is supported.
+ * \note           Only "dnsName", "uniformResourceIdentifier" and "otherName",
+ *                 as defined in RFC 5280, is supported.
```

Comment:
Ah, OK, I thought that the mistake was mentioning 4180 instead of 5280, but hardware module name is described in more detail in 4108, so that's another thing. I'll fix the comment.